src/core/tsi/ssl_transport_security.cc

```diff
@@ -1146,6 +1211,17 @@ static tsi_result ssl_handshaker_result_extract_peer(
       &peer->properties[peer->property_count]);
   if (result != TSI_OK) return result;
   peer->property_count++;
+
+  if (ca_cert != nullptr) {
+    result = peer_property_from_x509_subject(
+        ca_cert, &peer->properties[peer->property_count], true);
+    if (result != TSI_OK) {
+      gpr_log(GPR_ERROR, "resut: %d", static_cast<int>(result));
```

Comment:
nit: typo in "result". Also can we add a more informative error message here?

Reply:
We had placed this log here for debugging - do you think we actually need it? Above a lot of the properties follow the pattern `if result != TSI_OK) return result;`. I think the `peer_chain` is the only property above that doesn't return result.

I guess if we reach here, what we're saying is "there is a root cert that is set in the ex_data, but adding it as a property to `tsi_peer` is failing." So what's our desired behavior here?